### ISSUE TITLE
Hardcoded `-rc` argument for isort is depracated

### ISSUE THREAD

**Issue by abey79** [ricardochaves/python-lint#6]
This action hardcodes `-rc` as argument to isort, but this triggers a deprecation warning:

```
 $ isort -rc -c --diff tests
/Users/hhip/src/vsketch/venv/lib/python3.8/site-packages/isort/main.py:771: UserWarning: 

The following deprecated CLI flags were used: -rc!
Please see the 5.0.0 upgrade guide:
        https://timothycrosley.github.io/isort/docs/upgrade_guides/5.0.0/
```

I suggest removing it altogether, it's easy to re-add via `extra-isort-options` if needed.

**Comment by ricardochaves** [comment]
I created a release of Isort 5 v1.2.0. But Pylint is not working with isort 5. So a updated the README.md with this limitation and Ill follow the Pylint problem, when it's fixed I update the repo.
